app/uk/gov/hmrc/emcstfe/services/GetMovementService.scala

```diff
@@ -39,18 +40,23 @@ import scala.xml.{NodeSeq, XML}
 
 @Singleton
 class GetMovementService @Inject()(
-                                    connector: ChrisConnector,
+                                    chrisConnector: ChrisConnector,
+                                    eisConnector: EisConnector,
                                     repository: GetMovementRepository,
                                     xmlUtils: XmlUtils,
                                     val config: AppConfig,
-                                  ) extends Logging {
+                                  ) extends Logging with FeatureSwitching {
   def getMovement(getMovementRequest: GetMovementRequest, forceFetchNew: Boolean)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[ErrorResponse, GetMovementResponse]] = {
     repository.get(getMovementRequest.arc).flatMap {
       case Some(value) =>
         logger.info("[getMovement] Matching movement found")
         if (forceFetchNew) {
           logger.info("[getMovement] GetMovementIfChanged")
-          getMovementIfChanged(getMovementRequest, value)
+          if(isEnabled(SendToEIS)) {
+            getNewMovement(getMovementRequest)
+          } else {
+            getMovementIfChanged(getMovementRequest, value)
```

Comment:
In the new world this will no longer be valid and all calls will go through get new Movement